web/src/main/java/org/apache/shiro/web/filter/authc/FormAuthenticationFilter.java

```diff
@@ -1,227 +0,0 @@
-/*
```

Comment:
This file should be deleted

Reply:
can I make individual PRs for each changes, so that it will be easy for you and me to track the changes?

Reply:
Sorry, I am actually new to the open-source contribution, I am really excited to contribute to open-source projects starting with Apache Shiro.

Reply:
Thanks, that's great!
Please don't make individual PRs for changes. The first PR was *almost* good to go until you decided to put in some "controversial" changes last minute :)

Reply:
Can you help me why the license check is failing in the third pipeline?

Reply:
Let's take care of the comments first